### ISSUE TITLE
Actual hyperparameters for DrawerWorld benchmark

### ISSUE THREAD

**Issue by jiyeooong** [TonyLianLong/VAI-ReinforcementLearning#6]
hello, I experimented with your code using your DrawerWorld benchmark, but I can't get the performance written in your paper.

Can you tell me the hyperparameter for DrawerWorld you used in the actual experiment? I think it is different from DMC hypereparameter written in Readme.MD.
(I only changed action repeat to 2)

Thank you very much for your benchmark!

**Comment by TonyLianLong** [comment]
Hi @jiyeooong , could you provide more details? It is hard to infer what you encounter from the description.

1. Did you use our pretrained checkpoint and adapter, or do you train your own adapter or RL module? You probably want to tell me about how each step.
2. What did you observe in DrawerWorld? I remember DrawerWorld does not require much tuning, as its distraction is static. You should try without adapter and without distraction to see whether it works.
3. If DrawerWorld works without adapter but doesn't work with adapter, then we can look into the quality of adapter. A good way to see if your adapter works is to load images from DrawerWorld and visualize the mask from adapter and masked image to see if it sufficiently removes the background.
